scripts/consolidate-memory.py

```
@@ -0,0 +1,492 @@
+    elif group.score >= 0.4:
+        return 'REVIEW'
+    elif group.count >= 3:
+        return 'CANDIDATE'
+    elif group.count >= 2 and len(group.dates) >= 2:
+        return 'CANDIDATE'
+    elif group.count >= 2:
+        return 'REVIEW'
+    return 'REJECTED'
+
+
+def generate_memory_index(result: dict, memory_dir: Path) -> str:
```

Ruff (0.14.14)

[warning] Unused function argument: `memory_dir`

(ARG001)